demo/simple/sysl-app-hyperlink.sysl

```diff
@@ -0,0 +1,12 @@
+MobileApp:
+    @link = "www.foofle.com"
```

Comment:
Might want to add that this needs to have https:// trailing otherwise browsers think it's a relative link (just tried it now)

Reply:
Addressed

Reply:
Addressed